stdlib/json.gr

```diff
@@ -0,0 +1,2112 @@
+/**
+ * JSON (JavaScript Object Notation) parsing, printing, and access utilities.
+ *
+ * @example include "json"
+ * @example Json.parse("{\"currency\":\"€\",\"price\":99.99}")
+ * @example
+ * print(
+ *   toString(
+ *     format=Pretty,
+ *     JsonObject([("currency", JsonString("€")), ("price", JsonNumber(99.9))])
+ *   )
+ * )
+ */
+
+module Json
+
+include "runtime/bigint" as BI
+include "runtime/dataStructures"
+include "runtime/numbers"
+include "runtime/numberUtils"
+include "runtime/string" as RuntimeString
+include "runtime/unsafe/tags"
+include "runtime/unsafe/wasmi32"
+include "runtime/unsafe/wasmi64"
+include "runtime/unsafe/wasmf64"
+include "runtime/atof/parse" as Atof
+include "buffer"
+include "char"
+include "string"
+include "list"
+include "uint8"
+from RuntimeString use { toString as runtimeToString, getCodePoint }
+from Numbers use { coerceNumberToWasmI32 }
+from DataStructures use { tagSimpleNumber, untagSimpleNumber }
+
+// Primitive offsets
+// TODO(#703): Get these offsets from the runtime
+@unsafe
+let _INT64_BOXED_VALUE_OFFSET = 8n
+@unsafe
+let _Float64_BOXED_VALUE_OFFSET = 8n
```

Comment:
I thought I saw these exported from somewhere a little while ago. Can these be imported from the runtime yet?

Reply:
I don't see anywhere these are provided. We could certainly move them into the runtime in this pr though? Should they go in `runtime/constants` or `runtime/tags` though?